src/circuit/circuit_frame.rs

```diff
@@ -1479,6 +1481,25 @@ fn reduce_cons<F: PrimeField, CS: ConstraintSystem<F>>(
             &[env, &arg1, cont, &[&g.default_num, &g.default_num]],
         )?;
 
+        let call0_continuation = AllocatedContPtr::construct(
+            &mut cs.namespace(|| "Call0"),
+            store,
+            &g.call0_cont_tag,
+            &[
+                cont,
+                &[&g.default_num, &g.default_num],
+                &[&g.default_num, &g.default_num],
+                &[&g.default_num, &g.default_num],
+            ],
+        )?;
+
+        let the_call_continuation = AllocatedContPtr::pick(
+            &mut cs.namespace(|| "the_call_tag"),
```

Comment:
Namespace name should probably agree with variable name for clarity and general convention in such cases.